### ISSUE TITLE
Games won't load from playlists on Xbox Series S - Retail RA

### ISSUE THREAD

**Issue by jay81uk** [libretro/RetroArch#13346]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

Playlist set up in retail Xbox retroarch. Games will not load when selecting a game from playlists but will load when manually loading core and manually loading a game

### Expected behavior

Game to load when chosen from playlist

### Actual behavior

Black screen when loading game and RA crashes

### Steps to reproduce the bug

1. Import content
2. Select core, system name, folder where games are located, start scan
3. Open playlist and run a game - black screen crash

### Bisect Results

Multiple latest builds on RA site

### Version/Commit
You can find this information under Information/System Information

- RetroArch: MSVC (1929) 64 bit

### Environment information

- OS: Xbox Series S
- Compiler: MSVC (1929) 64 bit

**Comment by Gamr13** [comment]
Can confirm this issue. 

To add more detail, it happens after attempting to add multiple playlists, when the scanning process freezes at 0% the bug is triggered. If the user quits RetroArch and opens it back up and attempts to load a game from a playlist, there will be an infinite black screen. 

I replicated this issue by performing these exact steps:
1) Clean install of RetroArch
2) Import Content > Manual Scan
3) Set content directory, core, file extension and system name
4) Start Scan and created 4 playlists.

I then restarted RetroArch and attempted to add a 5th playlist using the above steps which froze at 0%. I attempted to Quit RetroArch using the button in Main Menu, which caused RetroArch to freeze and not quit, after a forced termination, I attempted to load a game in one of my newly created playlists, which hung on a black screen.
